setup-elytron-guides/setup_elytron_windows.adoc

```diff
@@ -0,0 +1,148 @@
+---
+layout: setup-elytron
+title: Setting up WildFly Elytron on Windows
+permalink: /guides/setup-elytron-windows/
+os: windows
+---
+:imagesdir: ../../assets/images/setup-elytron/
+
+:toc: macro
+:toc-title:
+
+This page will guide you through the steps needed to setup a development environment for WildFly Elytron on Windows.
+
+toc::[]
+
+== Installing Java
+
+. First, let's check if you already have Java installed.
+    .. Open Command Prompt, and run `java --version`.
+    .. If the output shows a version number, check to make sure it is Java 11 (e.g., "11.0.15.1"). If so, Java 11 is already installed and you can jump to the "Installing Maven" section. If the output is not Java 11, or if the output is `'java' is not recognized as an internal or external command, operable program or batch file`, continue with the following steps.
+        ... If you encounter a message such as `Java was not found, run without arguments to install from the Microsoft Store.`, follow https://www.windowscentral.com/how-manage-app-execution-aliases-windows-10[these instructions] to disable the alias.
+    .. Close the Command Prompt.
+. Go to https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html[this page] (or use your browser to search for "Download JDK 11") and download the "x64 Installer".
+. Run the file that you downloaded (installer shown below) and note where it is being installed.
++
+[.responsive-img_capped.text-center]
+image::setup-elytron_windows_install-jdk.png[Screenshot of Windows Oracle JDK installer]
+
+    .. Now open Command Prompt and run `java --version` once again. This time, the output should be something like (the exact version will depend on the version that was installed):
++
+[source,batch]
+----
+java version "11.0.15.1" 2022-04-22 LTS
+Java(TM) SE Runtime Environment 18.9 (build 11.0.15.1+2-LTS-10)
+Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.15.1+2-LTS-10, mixed mode)
+----
+    .. If you don't see the output above, that means you'll need to update your “Path”. To do so:
+        ... Search for “Edit the system environment variables” in your Start menu. 
+        ... Click on “Environment Variables”.
+        ... Now double click the “Path” under “User variables for `<Windows username>`”. Then click “New” and enter the directory from the installer with “\bin” at the end. In this example, we'd need to add _C:\Program Files\Java\jdk-11.0.15.1\bin_ to the “Path”. +
```

Comment:
@cam-rod The \bin is not rendered, it shows: `enter the directory from the installer with "" at the end`